packages/finer-wallet/src/lib/finer-wallet.ts

```diff
@@ -50,7 +52,7 @@ export function setupFinerWallet({
         description: null,
         iconUrl,
         deprecated,
-        available: isMobile(),
+        available: true,
```

Comment:
What will happen if Finer Wallet application is not installed and you try to connect?

Reply:
in mobile browser - nothing will happen

Reply:
I was thinking, we can make communication through the https:// scheme, and if the application is not installed, then the user will be taken to a website where he can proceed to install the application on the App Store

Reply:
Currently this is how it looks when an injected wallet is not installed.

![image](https://user-images.githubusercontent.com/95852708/189116078-cb3cf092-b28e-45a2-876f-8eeb03394129.png)

This screen will show if `available` is false. The Open button will redirect you to `downloadUrl` (your app download page)